terraform/eks/otlp/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "region" {
+  default = "us-west-2"
+}
+
+variable "testing_id" {
+  type = string
+}
+
+variable "eks_pod_config_path" {
```

Comment:
do we need a default value here?

Reply:
We could have one. I thought it was unnecessary because `/eks/otlp` is actually an inner module of `eks` (to isolate bootstrap process for otls, prometheus and infra), the var input is taken cared of by us, and is not exposed to end user.